test/test_model.py

```diff
@@ -361,7 +351,11 @@ def check_remote_units():
         # Create a new model and backend to drop a cached is-leader output.
         self.backend = ops.model.ModelBackend()
         meta = ops.charm.CharmMeta()
-        meta.relations = {'db0': None, 'db1': None, 'db2': None}
+        meta.relations = {
```

Comment:
Why was it ok before to have None (presumably it was populating it when it was needed, given the __init__ of Model also sets everything to None)?

Reply:
`RelationMeta` was not used in `model.py` for anything. There was no code that would access `self.relations['foo'].<attribute>`.